users/postgres/init.go

```diff
@@ -96,6 +96,13 @@ func migrateDB(db *sqlx.DB) error {
 				)`,
 				},
 			},
+			{
+				Id: "users_5",
+				Up: []string{
```

Comment:
Maybe you should move this to dedicated PR, so we keep the clean history (this one is important).
Let's follow an established rule for not refactoring the code or make changes, not related to a specific feature/fix that you are working on.